src/sass/messaging/partials/_messaging-folder.scss

```diff
@@ -105,10 +105,24 @@
         }
       }
 
-      // Date
+      // Attachment icon
       &:nth-child(3) {
         font-weight: normal;
         position: absolute;
+        right: 25%;
+        text-align: right;
+        top: 0;
+        width: 25%;
```

Comment:
I was trying to make it so that the widths of the From/To field and the Date field would add up to 100%. If this is to take up some space, the width of From/To or Date should be decreased. I would probably at least decrease the width of the Date field a bit so that the icon isn't too far away from it, while ensuring a long From/To doesn't mess up the layout.